### ISSUE TITLE
HACs hassio 1.0.3 show warning

### ISSUE THREAD

**Issue by HABuild** [iantrich/roku-card#27]
Hi Ian,

I updated this via HACs it initially didn't take the breaking changes updates, when changed in my config, perhaps it was cached. However now after a few restarts it doesn't show at all except for a single line box with show warning in tan color. 
I have tried uninstall/reinstall, and dropping your example config in also, gives the same result

edit. 
ok manual install of 1.0.1 works, dropping in 1.0.3 in the same directory and clearing the cache, gives the same show warning

![show_warning](https://user-images.githubusercontent.com/36288425/68083191-56f6f080-fe7a-11e9-8341-d12f6ce4f927.PNG)

````
type: 'custom:roku-card'
entity: media_player.basement_roku
tv: true
apps:
  - image: /local/icons/netflix.webp
    app: Netflix
  - image: /local/icons/emby.webp
    app: Hulu
Play:
  tap_action:
    action: call-service
    service: media_player.media_play_pause
    service_data:
      entity_id: media_player.chromecast_ultra
back:
  tap_action:
    action: call-service
    service: switch.turn_on
    service_data:
      entity_id: switch.ir_exit
down:
  tap_action:
    action: call-service
    service: switch.turn_on
    service_data:
      entity_id: switch.ir_down
home:
  tap_action:
    action: call-service
    service: switch.turn_on
    service_data:
      entity_id: switch.ir_menu
info:
  tap_action:
    action: call-service
    service: switch.turn_on
    service_data:
      entity_id: switch.ir_source
left:
  tap_action:
    action: call-service
    service: switch.turn_on
    service_data:
      entity_id: switch.ir_left
power:
  tap_action:
    action: call-service
    service: switch.turn_on
    service_data:
      entity_id: switch.ir_power
right:
  tap_action:
    action: call-service
    service: switch.turn_on
    service_data:
      entity_id: switch.ir_right
select:
  tap_action:
    action: call-service
    service: switch.turn_on
    service_data:
      entity_id: switch.ir_ok
up:
  tap_action:
    action: call-service
    service: switch.turn_on
    service_data:
      entity_id: switch.ir_up
volume_down:
  hold_action:
    action: call-service
    repeat: 500
    service: switch.turn_on
    service_data:
      entity_id: switch.ir_vol_dn
  show: true
  tap_action:
    action: call-service
    service: switch.turn_on
    service_data:
      entity_id: switch.ir_vol_up
volume_mute:
  hold_action:
    action: call-service
    repeat: 500
    service: switch.turn_on
    service_data:
      entity_id: switch.ir_vol_mute
  show: true
  tap_action:
    action: call-service
    service: switch.turn_on
    service_data:
      entity_id: switch.ir_vol_up
volume_up:
  hold_action:
    action: call-service
    repeat: 500
    service: switch.turn_on
    service_data:
      entity_id: switch.ir_vol_up
  show: true
  tap_action:
    action: call-service
    service: switch.turn_on
    service_data:
      entity_id: switch.ir_vol_up
````

**Comment by iantrich** [comment]
I made a change so that the media player entity's state object is required (obviously forgot to change the warning text however) Does `media_player.basement_roku` exist or are you trying to use this as a generic remote?

**Comment by iantrich** [comment]
For now, put an entity that exists in there as a workaround.

**Comment by iantrich** [comment]
I just realized that I inadvertently removed `repeat` functionality. So wait for 1.0.5

**Comment by HABuild** [comment]
Ahh. Yes I just use it as generic remote. I don't have a roku.

**Comment by iantrich** [comment]
fixed in 1.0.6

**Comment by HABuild** [comment]
thanks 1.06 works. I like the entity change too!
